src/hivpy/prep.py

```diff
@@ -21,9 +26,218 @@ class PrEPModule:
 
     def __init__(self, **kwargs):
         # FIXME: move these to data file
```

Comment:
I forgot to do this earlier, but I should probably address the data file FIXME as part of this PR.